src/api/crypto.js

```diff
@@ -54,18 +55,6 @@ class PubEncKey extends h.NetworkObject {
     return lib.encrypt_sealed_box(this.app.connection, str, this.ref);
   }
 
-  /**
-  * Decrypt the given cipher text (buffer or string) using this public
-  * encryption key and the given secret key
-  *
-  * @arg {Buffer} cipher to decrypt
-  * @arg {SecEncKey} secretEncKey secret encryption key
-  * @returns {Promise<Buffer>} plain text
-  */
-  decrypt(cipher, secretEncKey) {
```

Comment:
Why are we removing this function?

Reply:
I was confused about this part of the API. Why do `PubEncKey` and `SecEncKey` each have an `encrypt` and `decrypt` function?

I don't see that reflected in web API:
- https://github.com/joshuef/peruse/blob/master/app/extensions/safe/api/crypto_public_encryption_key.js#L3
- https://github.com/joshuef/peruse/blob/master/app/extensions/safe/api/crypto_secret_encryption_key.js#L3